checkQC/handlers/q30_handler.py

```diff
@@ -32,17 +32,21 @@ def check_qc(self):
             lane_nbr = int(error_dict["lane"])
             read = error_dict["read"]
             percent_q30 = error_dict["percent_q30"]
-
+            is_index_read = error_dict["percent_q30"]
+            read_or_index_text = "index read" if is_index_read else "read"
+            
             if self.error() != self.UNKNOWN and percent_q30 < self.error():
-                yield QCErrorFatal("%Q30 {:.2f} was too low on lane: {} for read: {}".format(percent_q30,
+                yield QCErrorFatal("%Q30 {:.2f} was too low on lane: {} for {}: {}".format(percent_q30,
                                                                                              lane_nbr,
+                                                                                             read_or_index_text,
                                                                                              read),
                                    ordering=lane_nbr,
                                    data={"lane": lane_nbr, "read": read,
                                          "percent_q30": percent_q30, "threshold": self.error()})
             elif self.warning() != self.UNKNOWN and percent_q30 < self.warning():
-                yield QCErrorWarning("%Q30 {:.2f} was too low on lane: {} for read: {}".format(percent_q30,
+                yield QCErrorWarning("%Q30 {:.2f} was too low on lane: {} for {}: {}".format(percent_q30,
                                                                                                lane_nbr,
+                                                                                               read_or_index_text,
```

Comment:
Github does not allow me to make a suggestion that spans all the lines here but please apply the same formatting than above

Reply:
I have now done this. Thanks for the reminder that if i can leave the file i am working on better, I should probably do that.